mxcube3/routes/scutils.py

```diff
@@ -152,11 +152,16 @@ def mount_sample_clean_up(sample):
 
         if sample['location'] != 'Manual':
             if not mxcube.sample_changer.getLoadedSample():
-                mxcube.sample_changer.load(sample['sampleID'], wait=False)
+                mxcube.sample_changer.load(sample['sampleID'], wait=True)
             elif mxcube.sample_changer.getLoadedSample().getAddress() != sample['location']:
-              mxcube.sample_changer.load(sample['sampleID'], wait=False)
+              mxcube.sample_changer.load(sample['sampleID'], wait=True)
+
         mxcube.shapes.clear_all()
 
+        if sample['location'] != 'Manual':
+            C3D_MODE = mxcube.diffractometer.C3D_MODE
+            mxcube.diffractometer.startCentringMethod(C3D_MODE)
```

Comment:
this should be dependent on the `auto_loop_centring`, not by default

Reply:
Its not the decision that we took here for the moment, it can definitely be argued but they prefer that its performed by default for the moment. The idea was the the auto_loop_centring options only applies to the queue, and that we have a consistent behavior for the rest. 

Does not really matter to me I can understand both opinions, in this case Ive let our users decide for the moment and we will see if it something that makes sense in general later on.

Reply:
okey, we have our sample changer operational yet so cannot say much against. Perhaps we could add later the option at init. Fine as it is now.